kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -132,6 +132,8 @@ case class KyuubiConf(loadSysDefault: Boolean = true) extends Logging {
   private val serverOnlyConfEntries: Set[ConfigEntry[_]] = Set(
     FRONTEND_BIND_HOST,
```

Comment:
Do we have any strategy for deprecated config options?

Reply:
Maybe you can refer to LEGACY_ENGINE_SHARE_LEVEL?

Reply:
use `fallbackConf`

Reply:
Got it